bench-vortex/src/clickbench.rs

```diff
@@ -0,0 +1,264 @@
+        Field::new("funiqid", Int64, false),
+        Field::new("originalurl", Utf8View, false),
+        Field::new("hid", Int32, false),
+        Field::new("isoldcounter", Int16, false),
+        Field::new("isevent", Int16, false),
+        Field::new("isparameter", Int16, false),
+        Field::new("dontcounthits", Int16, false),
+        Field::new("withhash", Int16, false),
+        Field::new("hitcolor", Utf8View, false),
+        Field::new(
+            "localeventtime",
+            Timestamp(TimeUnit::Microsecond, None),
+            false,
+        ),
+        Field::new("age", Int16, false),
+        Field::new("sex", Int16, false),
+        Field::new("income", Int16, false),
+        Field::new("interests", Int16, false),
+        Field::new("robotness", Int16, false),
+        Field::new("remoteip", Int32, false),
+        Field::new("windowname", Int32, false),
+        Field::new("openername", Int32, false),
+        Field::new("historylength", Int16, false),
+        Field::new("browserlanguage", Utf8View, false),
+        Field::new("browsercountry", Utf8View, false),
+        Field::new("socialnetwork", Utf8View, false),
+        Field::new("socialaction", Utf8View, false),
+        Field::new("httperror", Int16, false),
+        Field::new("sendtiming", Int32, false),
+        Field::new("dnstiming", Int32, false),
+        Field::new("connecttiming", Int32, false),
+        Field::new("responsestarttiming", Int32, false),
+        Field::new("responseendtiming", Int32, false),
+        Field::new("fetchtiming", Int32, false),
+        Field::new("socialsourcenetworkid", Int16, false),
+        Field::new("socialsourcepage", Utf8View, false),
+        Field::new("paramprice", Int64, false),
+        Field::new("paramorderid", Utf8View, false),
+        Field::new("paramcurrency", Utf8View, false),
+        Field::new("paramcurrencyid", Int16, false),
+        Field::new("openstatservicename", Utf8View, false),
+        Field::new("openstatcampaignid", Utf8View, false),
+        Field::new("openstatadid", Utf8View, false),
+        Field::new("openstatsourceid", Utf8View, false),
+        Field::new("utmsource", Utf8View, false),
+        Field::new("utmmedium", Utf8View, false),
+        Field::new("utmcampaign", Utf8View, false),
+        Field::new("utmcontent", Utf8View, false),
+        Field::new("utmterm", Utf8View, false),
+        Field::new("fromtag", Utf8View, false),
+        Field::new("hasgclid", Int16, false),
+        Field::new("refererhash", Int64, false),
+        Field::new("urlhash", Int64, false),
+        Field::new("clid", Int32, false),
+    ])
+});
+
+pub async fn register_vortex_file(
+    session: &SessionContext,
+    table_name: &str,
+    file: &Path,
+    schema: &Schema,
+    enable_compression: bool,
+) -> anyhow::Result<()> {
+    let vortex_dir = file.parent().unwrap().join(if enable_compression {
+        "vortex_compressed"
+    } else {
+        "vortex_uncompressed"
```

Comment:
No real point in uncompressed file?

Reply:
definetly some more cleanup to do here, some ironing out some bugs first.